charts/otomi-api/templates/clusterrole.yaml

```diff
@@ -24,7 +24,7 @@ metadata:
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: otomi-api
+  name: cluster-admin
```

Comment:
is it necessary to change this name?

Reply:
yes, basically we are assigning the cluster-admin clusterrole to the otomi-api service account so it can later create the needed role and clusterrolebindings/rolebindings for the tty service accounts.